data/json/professions.json

```
@@ -3130,6 +3137,7 @@
     "name": "Naturalist",
     "description": "Over long years of self-imposed exile in the wilderness, you have come to an understanding with Mother Nature.  The world as they knew it might have ended for your forsaken species, but you can hardly tell the difference.",
     "points": 6,
+    "proficiencies": [ "prof_fibers", "prof_fibers_rope" ],
```

There's a second proficiencies line after the skills on line 3153, need to delete that one to load in.
```suggestion
    "proficiencies": [ "prof_fibers", "prof_fibers_rope", "prof_knapping" ],
```